server/slack/webhooks.js

```diff
@@ -141,6 +130,28 @@ const webhooks = {
   listHangout(channel) {
     const count = HangoutHelper.getUpcomingHangoutCounts();
     SlackAPI.postMessage(channel, `${count} hangouts are scheduled currently.`);
+  },
+
+  createTodayILearned(slackUserId, action, channel) {
+    const slackUser = SlackAPI.getUser(slackUserId);
+    if (!slackUser) {
+      return console.error("slackWebhooks.processEvent[slack user not found]");
+    }
+    const slackUserEmail = slackUser.profile && slackUser.profile.email;
+    const user = Meteor.users.findOne({ email: slackUserEmail });
+    if (!user) {
+      console.error("slackWebhooks.processEvent[meteor user not found]");
+      return SlackAPI.postMessage(channel, "Your account was not found on codebuddies.org");
+    }
+
+    LearningHelper.addLearning({
+      title: action.topic,
+      user_id: user._id,
+      username: user.username
+    });
+
+    const url = Meteor.absoluteUrl("/hangouts");
+    SlackAPI.postMessage(channel, `Today I learned is created for you at ${url}.`);
```

Comment:
Could we change the message to `Your learning was archived on ${url}`?

Reply:
sure

Reply:
@lpatmo  fixed